draft-ietf-quic-http.md

```diff
@@ -2572,7 +2576,8 @@ for what might be a temporary or intermittent error.
 # Acknowledgments
 {:numbered="false"}
 
-The original authors of this specification were Robbie Shade and Mike Warres.
+Robbie Shade and Mike Warres were the authors of draft-shade-quic-http2-mapping,
+which was the basis for this document.
```

Comment:
RFC Editor asked:
> We believe "original authors of this specification" refers to the authors of draft-shade-quic-http2-mapping; may we clarify that here?

Reply:
Perhaps this can be less generous, as "basis" implies a lot more substance was provided by that document.

```suggestion
Robbie Shade and Mike Warres were the authors of draft-shade-quic-http2-mapping,
a precursor of this document.
```